feedthefox/base/static/js/profile.js

```diff
@@ -4,8 +4,33 @@ jQuery(document).ready(function ($) {
     var t = $('#id_device_info-imei');
     t.bind('propertychange keyup input paste', function(event) {
         var target = $('.imei-tos');
+        var status = $('.imei-status');
         var term = t.val();
-        (term !== '' ? $(target).show('fast') : $(target).hide('fast'));
+        if (term !== '') {
+            $(target).show('fast');
+            $(status).show();
+            $.ajax({
+                url: "/imei/" + term,
+                type: "GET",
+                success : function(json) {
+                    if (json.status) {
+                        $(status).removeClass('fa-times');
+                        $(status).addClass('fa-check');
+                        $(status).text(' valid');
```

Comment:
Instead of using text and check mark ,maybe we could just change the color of the input box. Keep it blue when it is valid and change it to red while it's invalid.